content/en/tracing/connect_logs_and_traces/java.md

```diff
@@ -69,22 +81,28 @@ import datadog.trace.api.CorrelationIdentifier;
 
 // there must be spans started and active before this block.
 try {
+    MDC.put("dd.env", String.valueOf(CorrelationIdentifier.getEnv()));
+    MDC.put("dd.service", String.valueOf(CorrelationIdentifier.getService()));
+    MDC.put("dd.version", String.valueOf(CorrelationIdentifier.getVersion()));
     MDC.put("dd.trace_id", String.valueOf(CorrelationIdentifier.getTraceId()));
     MDC.put("dd.span_id", String.valueOf(CorrelationIdentifier.getSpanId()));
 }
 
 // Log something
 
 finally {
+    MDC.remove("dd.env");
+    MDC.remove("dd.service");
+    MDC.remove("dd.version");
     MDC.remove("dd.trace_id");
     MDC.remove("dd.span_id");
 }
 ```
 
-Then update your logger configuration to include `dd.trace_id` and `dd.span_id` in your log pattern:
+Then update your logger configuration to include `dd.env`, `dd.service`, `dd.version`, `dd.trace_id`, and `dd.span_id` in your log pattern:
 
 ```xml
-<Pattern>"%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %X{dd.trace_id:-0} %X{dd.span_id:-0} - %m%n"</Pattern>
+<Pattern>"%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %X{dd.env} %X{dd.service} %X{dd.version} %X{dd.trace_id:-0} %X{dd.span_id:-0} - %m%n"</Pattern>
```

Comment:
Same here, let's review the pattern, it is the opportunity to move to a keyvalue format to be similar to other languages (and potentially quote around the span and trace ids